source/server/hot_restart_impl.cc

```diff
@@ -84,8 +85,8 @@ SharedMemory& SharedMemory::initialize(uint32_t stats_set_size, Options& options
   }
 
   // Stats::RawStatData must be naturally aligned for atomics to work properly.
-  RELEASE_ASSERT((reinterpret_cast<uintptr_t>(shmem->stats_set_data_) % alignof(RawStatDataSet)) ==
-                 0);
+  RELEASE_ASSERT(
```

Comment:
What changed here?

Reply:
`RawStatDataSet` -> `Stats::RawStatDataSet`, which pushed it over some character limit and reflowed the formatting.

Reply:
Ah, feel free to disregard.